apps/nsqlookupd/main.go

```diff
@@ -0,0 +1,22 @@
+// +build !windows
+
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func main() {
+	daemon := start()
```

Comment:
in the `main_windows.go` file we called this var `nsqd`, which I think I prefer.